shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -753,6 +810,46 @@ enum RewriteTableTy {
     Peers,
 }
 
+fn is_prepared_message(message: &mut Message) -> Option<(Vec<u8>, PreparedMetadata)> {
+    if let Some(Frame::Cassandra(CassandraFrame {
+        operation: CassandraOperation::Result(CassandraResult::Prepared(prepared)),
+        ..
+    })) = message.frame()
+    {
+        return Some((
+            prepared.id.clone().into_bytes().unwrap(),
```

Comment:
Can't unwrap here as a misbehaving server could send something that gets parsed into a None.
`.ok_or_else(|| anyhow!("result prepared contained empty id")?` should be enough.